src/tapis-app/Files/_components/Toolbar/TransferModal/TransferModal.tsx

```diff
@@ -29,6 +30,13 @@ const TransferModal: React.FC<ToolbarModalProps> = ({
   const [transfer, setTransfer] = useState<Files.TransferTask | null>(null);
   const { selectedFiles } = useFilesSelect();
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      focusManager.setFocused(true);
```

Comment:
Instead of calling focusManager is it possible to call the refetch hook from the useList for transfers? Right now it's refetching all listings, including systems, files, etc.

Reply:
It is. called refetch is setInterval callback